HARK/algos/egm.py

```diff
@@ -0,0 +1,125 @@
+
+
+def analytic_pi_star_y(self,
```

Comment:
'acted'
'retro-policy'

$\pi(x)$
$\pi_y(y)$

What to call this?

Reply:
how about 'chosen'